test/functional/wallet_resendwallettransactions.py

```diff
@@ -81,30 +84,50 @@ def run_test(self):
         #
         # So we will create the child transaction, use listreceivedbyaddress to see what the
         # ordering of mapWallet is, if the child is not before the parent, we will create a new
-        # child (via bumpfee) and remove the old child (via removeprunedfunds) until we get the
+        # child (via nSequence-=1 and fee_delta+=200) and remove the old child (via removeprunedfunds) until we get the
         # ordering of child before parent.
-        child_txid = node.send(outputs=[{addr: 0.5}], options={"inputs": [{"txid":txid, "vout":0}]})["txid"]
+        # It is not possible to use bumpfee, as in very unlikely cases
+        # repeatedly calling it will overflow the maximum allowed -maxtxfee or
+        # run the wallet out of funds.
```

Comment:
```suggestion
        # RBF the child using prioritisetransaction instead of adding fees because, in very unlikely cases,
        # it will overflow the maximum allowed -maxtxfee or
        # run the wallet out of funds.
```